utils/src/mdns.rs

```diff
@@ -11,6 +13,7 @@ use crate::networking::my_ipv4_addrs;
 pub fn advertise_service(
     service_name: &str,
     port: u16,
+    instance_id: &Uuid,
```

Comment:
Code Zamboni may be my new favourite term of art. I am very happy to benefit from your cleanup services, because it's good to see alternative approaches of things while I'm still in the learning zone.